docs/UserGuide.md

```diff
@@ -252,15 +260,19 @@ Sample Output:
 ---
 ### Searching for an applicant : `search`
 
-Allows you to search for applicants containing specified keywords, these keywords could be in **any** of the 
-applicant's name, job, round or skills.
+Allows you to search for applicants containing the input keywords and the keyword has to be an exact word. You can search for the applicant based on 
+name, job, round or skills. You can also search for a combination of keywords and the applicant with the most matching keywords will be on the top of the list.
```

Comment:
I think u can move this to the bottom part with multiple dashes, and give an example to illustrate.